fiftyone/server/db/_mongo_lightning.py

```
@@ -0,0 +1,627 @@
+            ),
+            _first(
+                field_path,
+                dataset,
+                -1,
+                is_frame_field,
+                limit=path.max_documents_search,
+            ),
+            _match(field_path, None, limit=path.max_documents_search),
+        ]
+
+        def _resolve_int(results):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_resolve_int`

(ANN202)